__tests__/index.js

```diff
@@ -82,29 +81,29 @@ const invalidCss = (
 
 `)
 
-test("no warnings with valid css", t => {
+it("Shows no warnings with valid css", () => {
```

Comment:
To this is lame, I change `S` to `s` and the [tests fail](https://gist.github.com/ntwb/5b845b34f450c8ea869c616acbb060ae) /shrug